### PR TITLE
Add item OpenCV in User Guide

### PR DESCRIPTION
**Pull request by TOsmanov**
### Motivation and context

This PR contains adding item OpenCV and images.
Also updated item controls sidebar and shortcuts
The following measures were taken to solve problem #2765
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/documentation/user_guide.md [hunk 1]
```diff
@@ -1138,6 +1140,40 @@ Detectors are used to automatically annotate one frame. Supported DL models are
 - This action will automatically annotates one frame.
   In the [Automatic annotation](#automatic-annotation) section you can read how to make automatic annotation of all frames.
 
+## OpenCV
+
+This tools based on Open CV, it is used to create polygons. To work with tool, you need to download it.
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:1145]: The tool based on [Open CV](https://opencv.org/) Computer Vision library which is an open-source product that includes many CV algorithms. Some of these algorithms can be used to simplify the annotation process.

Comment (nmanovic) [reply cvat/apps/documentation/user_guide.md:1145]: First step to work with OpenCV is to load it into CVAT. Click on the toolbar icon, then click `Load OpenCV`.

File: cvat/apps/documentation/user_guide.md [hunk 2]
```diff
@@ -1138,6 +1140,40 @@ Detectors are used to automatically annotate one frame. Supported DL models are
 - This action will automatically annotates one frame.
   In the [Automatic annotation](#automatic-annotation) section you can read how to make automatic annotation of all frames.
 
+## OpenCV
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:1143]:
```suggestion
## OpenCV tools
```

File: cvat/apps/documentation/user_guide.md [hunk 3]
```diff
@@ -1138,6 +1140,40 @@ Detectors are used to automatically annotate one frame. Supported DL models are
...
+
+This tools based on Open CV, it is used to create polygons. To work with tool, you need to download it.
+
+To load OpenCV, click on the toolbar icon, then click `Load OpenCV`.
+
+  ![](static/documentation/images/image198.jpg)
+
+Once it is loaded, the tool's functionality will be available.
+
+### Drawing
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:1153]:
```suggestion
### Intelligent scissors 
```

File: cvat/apps/documentation/user_guide.md [hunk 4]
```diff
@@ -1138,6 +1140,40 @@ Detectors are used to automatically annotate one frame. Supported DL models are
...
+
+Drawing is an AI method of creating a polygon by placing points with automatic drawing of a line between them.
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:1155]:
```suggestion
Intelligent scissors is a CV method of creating a polygon by placing points with automatic drawing of a line between them.
```

File: cvat/apps/documentation/user_guide.md [hunk 5]
```diff
@@ -26,6 +26,7 @@
   - [Track mode (advanced)](#track-mode-advanced)
   - [Attribute annotation mode (advanced)](#attribute-annotation-mode-advanced)
   - [AI Tools](#ai-tools)
+  - [OpenCV](#opencv)
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:29]:
```suggestion
  - [OpenCV](#opencv-tools)
```

File: cvat/apps/documentation/user_guide.md [hunk 6]
```diff
@@ -1138,6 +1140,40 @@ Detectors are used to automatically annotate one frame. Supported DL models are
...
+
+The tool based on [Open CV](https://opencv.org/) Computer Vision library which is an open-source product that includes many CV algorithms. Some of these algorithms can be used to simplify the annotation process.
+
+First step to work with OpenCV is to load it into CVAT. Click on the toolbar icon, then click `Load OpenCV`.
+
+  ![](static/documentation/images/image198.jpg)
+
+Once it is loaded, the tool's functionality will be available.
+
+### intelligent scissors
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:1153]:
```suggestion
### Intelligent scissors
```

File: cvat/apps/documentation/user_guide.md [hunk 7]
```diff
@@ -1138,6 +1140,40 @@ Detectors are used to automatically annotate one frame. Supported DL models are
 - This action will automatically annotates one frame.
   In the [Automatic annotation](#automatic-annotation) section you can read how to make automatic annotation of all frames.
 
+## OpenCV tools
+
+The tool based on [Open CV](https://opencv.org/) Computer Vision library which is an open-source product that includes many CV algorithms. Some of these algorithms can be used to simplify the annotation process.
```
Comment (bsekachev) [inline cvat/apps/documentation/user_guide.md:1145]: OpenCV should be written together

**Review by nmanovic — APPROVED** [review]

**Pull request merged**